lib/hound/helpers/log.ex

```diff
@@ -16,12 +18,20 @@ defmodule Hound.Helpers.Log do
   Fetches all console.log() lines of type 'WARINING' and CRITICAL, from the browser as a single string of log-lines.
   """
   def fetch_errors() do
+    fail_if_webdriver_selenium("fetch_errors()")
+    
     get_browser_log()
     |> Enum.filter(&(is_error(&1)))
     |> Enum.map(&(Map.get(&1, "message")))
     |> Enum.join("\n")
   end
 
+  defp fail_if_webdriver_selenium(function) do
+    if false == Enum.any?(["phantomjs", "chrome_driver"], fn(x) -> x == System.get_env("WEBDRIVER") end) do
```

Comment:
I is not a good idea to rely on the environment variable here, as the driver can be set in the config.
You can get the current driver with `Hound.driver_info`, so I suggest you change this to something like

```elixir
if match?({:ok, %{driver: "selenium"}, Hound.driver_info)
```